meilisearch-http/src/extractors/authentication/mod.rs

```diff
@@ -204,12 +206,17 @@ pub mod policies {
                 return None;
             }
 
+            let mut validation = TENANT_TOKEN_VALIDATION.clone();
+            validation.insecure_disable_signature_validation();
```

Comment:
are we sure about that too? The doc says that we should only do that if know what we are doing. If we really do, may be we should add a comment explaining why this is ok.

Reply:
Same as above. @ManyTheFish can you review this PR and especially the two parts @MarinPostma quoted here, please?

Reply:
Yes, here we don't already know the API key to decode properly the token, to do it, we have to get the key prefix stored in the token.
So:
1) We decode the token to get the prefix
2) We retrieve the API key from the prefix
3) We verify the token with the retrieved API key